src/main/Tree.razor

```diff
@@ -38,7 +38,10 @@
         </button>
         <div class="col p-0">
             <input class="form-control form-control-sm ml-1 rounded inputEditor" id="AvatarUrl" placeholder="Avatar URL" @bind="AvatarUrl" @onkeyup="@this.OnKeyPress" />
-            <button class="inputEditorButton" @onclick="this.ShowAvatarEditorBox"><i class="fas fa-edit"></i></button>
+            @if (AvatarUrl != null && AvatarUrl.Contains("cortex/neurons"))
+            {
+                <button class="inputEditorButton" @onclick="this.ShowAvatarEditorBox"><i class="fas fa-edit"></i></button>
```

Comment:
Are we able to just "disable" (grey out) the button instead of hiding it so the user is aware of the functionality somehow when the AvatarUrl is invalid?